.planning/milestones/v1.0-phases/03-sme-creator-agent/03-PATTERNS.md

```
@@ -0,0 +1,531 @@
+
+**Process:** {PROCESS_NAME}
+**Output:** `.planning/smes/{PROCESS_NAME}-SME.md`
+**Findings:** {blocker_count} BLOCKERs, {warning_count} WARNINGs, {watch_count} WATCHes
+**Commit:** {last_analyzed_commit}
+```
+</step>
+```
+
+---
+
+#### Critical rules block pattern
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)